### PR TITLE
Add package pyads

### PR DESCRIPTION
**Pull request by stlehmann**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/pyads/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  version: {{ version }}
+
+source:
+  url: https://github.com/stlehmann/{{ name }}/releases/download/{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ce00b9875a98f1d98448afb6be81978d5c37736de64ba9b9a432267a455fcbb7
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<36]
```
Comment (scopatz) [inline recipes/pyads/meta.yaml]: Can this be noarch python?

File: recipes/pyads/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - pyads  # [not win]
```
Comment (scopatz) [inline recipes/pyads/meta.yaml]: You can remove the selector here.  noarch packages only need to pass on Linux

File: recipes/pyads/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<36]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - python
```
Comment (scopatz) [inline recipes/pyads/meta.yaml]: for noarch, this should be `python >=3.6`

File: recipes/pyads/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    - pip
+  run:
+    - python
```
Comment (scopatz) [inline recipes/pyads/meta.yaml]: this should be `python >=3.6`

File: recipes/pyads/meta.yaml (hunk 1 above)
Comment (stlehmann) [reply recipes/pyads/meta.yaml]: There is a C++ library that needs to be built on Linux / OS X patforms but not on Windows. So I assumed noarch python is not an option...?

Comment (stlehmann) [reply recipes/pyads/meta.yaml]: Shall I still change to noarch python?

Comment (stlehmann) [reply recipes/pyads/meta.yaml:15]: @scopatz What can I do to make progress with this PR? I still don't think noarch would be the right choice here because there is a C library that needs to be built on Linux. Not so on Windows because there it relies on 3rd party software. I appreciate any help to make progress with this PR :)

**Comment by klauer** [comment]
@conda-forge/staged-recipes, requesting help on behalf of @stlehmann, as having this package on conda-forge would be a big benefit to its users. Thanks in advance :)

**Comment by chrisburr** [comment]
@conda-forge-admin, please restart ci

File: recipes/pyads/meta.yaml (hunk 2 above)
Comment (chrisburr) [inline recipes/pyads/meta.yaml:28]: Why doesn't this get tested on Windows? There needs to be some kind of test if only to make sure the package isn't empty.

Comment (klauer) [reply recipes/pyads/meta.yaml:28]: The functionality on platforms differs here:
* Windows: pyads uses separately-installed vendor-copyrighted `.dll` library (not packaged)
* Linux/OSX: pyads compiles and uses its own open-source equivalent

End result on build server:
* Windows: import test fails; dll not found
* Linux/OSX: import succeeds

What do you think is a good forward here?

Comment (stlehmann) [reply recipes/pyads/meta.yaml:28]: @chrisburr as @klauer wrote a function-test on Windows is not possible because of a missing vendor-library. However I could modifiy the test to check the operating system and only test for the package on Windows while doing full checks on Linux. Would this suffice?

Comment (stlehmann) [reply recipes/pyads/meta.yaml:28]: This turns out to be a problem. When importing pyads on Windows without the vendor software installed I will get an error because a library is missing.

**Comment by klauer** [comment]
@conda-forge-admin, please ping team

@stlehmann is looking for input as to how to resolve the concerns of the conda-forge team here. Are there any agreeable solutions given the platform-specific packaging uniqueness?

**Comment by stlehmann** [comment]
Sadly there doesn't seem to be any kind of reaction from the reviewers on suggestions made by the package maintainers to fix the problems. As one can see in the comments we are more then willing to fix any problems. But there needs to be some kind of dialog here. I know everyone is busy but I don't think it is good practice if the reviewers don't answer at all to questions that arise during the review process. Please keep in mind that this is not the main battleground for the package maintainers so we might need some help to get this package on conda-forge.

So maybe a reviewer has some time to spare and gives this PR a second look so this PR won't get auto-closed in another month :(

File: recipes/pyads/meta.yaml (hunk 1 above)
Comment (mariusvniekerk) [reply recipes/pyads/meta.yaml:15]: Can't be noarch due to cxx portion

**Comment by mariusvniekerk** [comment]
It looks okay to me.  Can we rebase this one to the latest master just to make sure it still builds?

**Review by mariusvniekerk — APPROVED** [review]

**Comment by stlehmann** [comment]
> It looks okay to me. Can we rebase this one to the latest master just to make sure it still builds?

@mariusvniekerk Thanks for taking care of us. I just rebased on the current master branch..

**Comment by chrisburr** [comment]
@stlehmann I'm very sorry for this getting lost, if it happens again feel free to ping the teams more often or ask on gitter (see the README for a link).

LGTM 👍

**Comment by stlehmann** [comment]
@chrisburr no worries. Thanks for taking care of it at last. Happy to have another package on conda-forge👍 😄
